keyboards/keychron/q0/q0.c

```diff
@@ -16,6 +16,20 @@
 
 #include "quantum.h"
 
+#ifdef ENCODER_ENABLE
+bool encoder_update_kb(uint8_t index, bool clockwise) {
+    if (!encoder_update_user(index, clockwise)) { return false; }
+    if (index == 0) {
```

Comment:
Just a heads up, if there is only every one encoder, no need to check the index.

Reply:
Resolved in recent commit https://github.com/qmk/qmk_firmware/pull/20307/commits/b46fadf19d3921c6f12b35b4a289053c8006fde0.